[docs] Show font smoothing override (#21057)
mui-org/material-ui: docs/src/pages/customization/globals/globals.md

For instance, you can have a look at the [Button](/api/button/#css).
Alternatively, you can always have a look at the [implementation](https://github.com/mui-org/material-ui/blob/master/packages/material-ui/src/Button/Button.js).

## Global CSS

> In the event you are using the [CssBasline](/components/css-baseline/) component to apply global resets, you can reuse the infrastructure to apply global styles. For instance:

Reviewer: ```suggestion
In the event you are using the [CssBaseline](/components/css-baseline/) component to apply global resets, you can reuse the infrastructure to apply global styles. For instance:
```

Reviewer: ```suggestion
If you are using the [CssBaseline](/components/css-baseline/) component to apply global resets, you can also use it to apply global styles. For instance:
```
I tried to simplify the sentence but it sounds wonky to me. Tbh it might not need it at all.
cc @mbrookes

Reviewer 2: Yep, my first instinct on reading it was to make the same change for "In the event". 

I have no preference between "you can also use it" and "it can also be used"...

Reviewer: I prefer "it can also be used" :)
